### ISSUE TITLE
"sassOptions should be moved to your ember-cli-build" deprecation warning should be removed

### ISSUE THREAD

**Issue by minusfive** [adopted-ember-addons/ember-cli-sass#171]
When used as a nested addon, and `sassOptions` are passed through the `config()` addon hook, this deprecation warning is still shown, even though it seems like the correct way of doing it, no?

**Comment by aexmachina** [comment]
I'm not working with Ember at the moment, so I can't really say. PRs are welcome though. @stefanpenner might be able to advise?

**Comment by hilljh82** [comment]
@deanpapastrat I agree. I am the author of [ember-cli-mdc](https://github.com/onehilltech/ember-cli-mdc). One of the reasons I wrote my own `sass` wrapper for [ember-cli-mdc](https://github.com/onehilltech/ember-cli-mdc) several years ago was because of this very issue. I have decided to migrate back to ember-cli-sass, and this behavior still exists. It would be great if this warning could be removed when using `ember-cli-sass` from within an add-on.
